modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -325,7 +323,7 @@ public void testAddObjectActionWithConditionExpression() throws Exception {
 			"equals(firstName, \"João\")", RandomTestUtil.randomString(),
 			RandomTestUtil.randomString(),
 			ObjectActionExecutorConstants.KEY_GROOVY,
-			ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE,
+			ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE,
```

Comment:
@marcelabc I'm a little worried about this. Does this mean that `ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE` will no longer work like it used to?